libs/wingsdk/src/core/app.ts

```diff
@@ -91,9 +103,12 @@ export class CdktfApp extends Construct implements IApp {
 
     super(cdktfStack, "Default");
 
+    this.pluginManager = new PluginManager(props.plugins ?? []);
```

Comment:
I am not super happy with the fact that this is only for `CdktfApp`... We need to do some architectural refactoring at the app level at some point to generalize this a bit.

(not for now).